mlflow/pyfunc/__init__.py

```diff
@@ -108,6 +110,7 @@ def add_to_model(model, loader_module, data=None, code=None, env=None):
     :return: Updated model configuration.
     """
     parms = {MAIN: loader_module}
+    parms[PY_VERSION] = PYTHON_VERSION
```

Comment:
Could we add a unit test to confirm the python version is in fact embedded into the model?